src/com/tutorial/main/Window.java

```diff
@@ -1,31 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Canvas;
-import java.awt.Dimension;
-
-import javax.swing.JFrame;
-
-public class Window extends Canvas{
-
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = -240840600533728354L;
```

Comment:
This should be consistent with changed revision. Since this is the first revision of it, it can be `-1L` or `1L`. The next time this class is changed, update it to `(-)2L`. This helps to ensure the serialized version is the same on both ends serializing and deserializing. Again though, don't think you need to worry about it for now.